server/config/game/specialists.json

```diff
@@ -129,7 +129,7 @@
             "id": 7,
             "key": "forward-field",
             "name": "Hyperspace Transport",
-            "description": "Increased carrier speed by x2. Deducts -1 Weapons from your side in combat and -1 Hyperspace Range from the carrier.",
+            "description": "Harnesses the energies of void-fold continuums to amplify vessel propulsion by a factor of 2. In this high-velocity mode, weapon array efficiency diminishes by -3. However, the astral jump radius of the carrier expands by +2 hyperspace range.",
```

Comment:
I understood "Amplify Vessel Propulsión" to mean that it speeds up the carrier, but it seems it is extends the range. 
In Universe, I feel we are going towards 

Speed = Engine strength.
Range = Fuel available before next refuel.

You also say " High velocity Mode", which _really_ sounds like it's gonna go faster 
Could we rephrase this to avoid player confusion?

Reply:
I agree